diff --git a/makefiles/git.mk b/makefiles/git.mk
--- a/makefiles/git.mk
+++ b/makefiles/git.mk
@@ -8,6 +8,7 @@ DEB_GIT_V   ?= $(GIT_VERSION)
 
 GIT_ARGS += uname_S=Darwin \
 	HOST_CPU=$(GNU_HOST_TRIPLE) \
+	SHELL_PATH=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin/sh \
 	DESTDIR=$(BUILD_STAGE)/git \
 	MANDIR=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man \
 	PYTHON_PATH=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin/python3 \
